viewer/src/utilities/networking/CdfModelDataClient.ts

```diff
@@ -90,6 +90,23 @@ export class CdfModelDataClient
     return createModelTransformation(modelMatrix);
   }
 
+  public async getModelCamera(modelIdentifier: {
+    modelId: number;
+    revisionId: number;
+    format: File3dFormat | string;
+  }): Promise<CameraConfiguration | undefined> {
+    const { modelId, revisionId } = modelIdentifier;
+    const model = await this.client.revisions3D.retrieve(modelId, revisionId);
+    if (model.camera !== undefined && model.camera.position !== undefined && model.camera.target !== undefined) {
```

Comment:
I understand that probably we have a stable API which either sets a value or undefined, but still if it doesn't really matter, I suggest checking for truthness here, not undefined. Because if API will send null instead of undefined that condition will be incorrectly evaluated as true because `null !== undefined` is true.

```suggestion
    if (model.camera && model.camera.position && model.camera.target) {
```

also it's just more readable.